components/airtable/new-or-modified-records-in-view.js

```diff
@@ -51,6 +51,12 @@ module.exports = {
         record.type = "record_modified"
         modifiedRecords++
       }
+
+      record.metadata = {}
```

Comment:
```
const { baseId, tableId, viewId } = this
record.metadata = {
  baseId,
  tableId,
  viewId
}
```

Technically this will be the same for all events, so you can also move this code up, defining a `metadata` object once above the `for` loop, and then just set `record.metadata = metadata` here.